Makefile

```
@@ -16,70 +9,56 @@ all: docs packages
 tmp:
 	mkdir -p $(TMP)/.fmf
 
-
 # Run the test suite, optionally with coverage
 test: tmp
-	python3 -m pytest -vvv -ra --showlocals tests/unit
+	hatch run test:unit
 smoke: tmp
-	python3 -m pytest -vvv -ra --showlocals tests/unit/test_cli.py
-coverage: tmp
-	coverage run --source=tmt,bin -m pytest -vvv -ra --showlocals tests
-	coverage report
-	coverage annotate
+	hatch run test:smoke
+coverage: tmp nitrateconf
+	hatch run test:cov
```

```suggestion
	hatch run test:coverage
```

Let's keep the names consistent.